packages/calcite-components/src/components/accordion-item/accordion-item.tsx

```diff
@@ -243,12 +241,35 @@ export class AccordionItem implements ConditionalSlotComponent {
 
   @Listen("calciteInternalAccordionChange", { target: "body" })
```

Comment:
maybe this one should be targeted to document for consistency? 👍

Reply:
or vice versa?

Reply:
I'll create an issue to audit all listeners not targeting the element itself and possibly define some guidelines. I think `document` makes sense for a direct custom event bus where we don't need capturing nor bubbling.